docs/pages/connect-your-client/tsh.mdx

```diff
@@ -599,6 +599,13 @@ scp"` and use the familiar syntax:
 $ scp -P 61122 -r files root@node:/path/to/dest
 ```
 
+Starting from Teleport 11 `tsh scp` always uses the SFTP protocol under
+the hood, so no flags need to be passed to enable this behavior.
+
+Starting from Teleport 10 the SCP and SFTP protocols are both supported by Nodes.
```

Comment:
1. I talked with @alexfornuto as we think that "Server Access" sounds better
2. I forgot that we ported the SFTP server changes to v9 too.

```suggestion
Starting from Teleport 9 the SCP and SFTP protocols are both supported by Server Access.
```